cypress/integration/checkout/checkout.basic.spec.js

```diff
@@ -1,54 +1,122 @@
-            title: `Pre-checkout instance ${Number(new Date())}`,
+            instanceTypeId: itemData.instanceTypeId,
+            title: itemData.instanceTitle,
           },
-          holdings: [{
-            holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
-            permanentLocationId: Cypress.env('locations')[0].id,
-            sourceId: source.id,
-          }],
+          holdings: [
+            {
+              holdingsTypeId: itemData.holdingTypeId,
+              permanentLocationId: defaultLocation.id,
+            },
+          ],
           items: [
-            [{
-              barcode: ITEM_BARCODE,
-              missingPieces: '3',
-              numberOfMissingPieces: '3',
+            {
+              barcode: itemData.barcode,
               status: { name: 'Available' },
-              permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
-              materialType: { id: Cypress.env('materialTypes')[0].id },
-            }],
+              permanentLoanType: { id: itemData.loanTypeId },
+              materialType: { id: itemData.materialTypeId },
+            },
           ],
         });
+      })
+      .then((specialInstanceIds) => {
+        itemData.testInstanceIds = specialInstanceIds;
       });
-  });
 
-  it('Basic flow', function () {
-    cy.visit('/checkout');
+    cy.createTempUser(userData.permissions)
+      .then((userProperties) => {
+        userData.username = userProperties.username;
+        userData.password = userProperties.password;
+        userData.userId = userProperties.userId;
+        userData.barcode = userProperties.barcode;
+        userData.firstName = userProperties.firstName;
+      })
+      .then(() => {
+        UserEdit.addServicePointViaApi(servicePoint.id, userData.userId, servicePoint.id);
+
+        cy.login(userData.username, userData.password);
+      });
+  });
 
-    cy.checkOutItem(Cypress.env('users')[0].barcode, ITEM_BARCODE);
+  after('Delete New Service point, Item and User', () => {
+    cy.visit('/checkin');
+    CheckInActions.checkInItemGui(itemData.barcode);
+    CheckInPane.checkResultsInTheRow(checkInResultsData);
+    InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(itemData.barcode);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [servicePoint.id]);
+    Users.deleteViaApi(userData.userId);
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      defaultLocation.institutionId,
+      defaultLocation.campusId,
+      defaultLocation.libraryId,
+      defaultLocation.id
+    );
+    ServicePoints.deleteViaApi(servicePoint.id);
+  });
 
-    cy.verifyItemCheckOut();
+  it('C356772 An active user with barcode can Check out item (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {
+    cy.visit('/checkout');
+    CheckOutActions.checkIsInterfacesOpened();
+    CheckOutActions.checkOutUser(userData.barcode);
+    CheckOutActions.checkUserInfo(userData, 'staff');
```

Comment:
Are you sure that you always have the patron group for the user with the name 'staff'? To be sure you need add this parameter during the creation of the temp user.